recipes/protobuf/all/conandata.yml

```diff
@@ -11,6 +11,9 @@ sources:
   3.13.0:
     sha256: 9b4ee22c250fe31b16f1a24d61467e40780a3fbb9b91c3b65be2a376ed913a1a
     url: https://github.com/protocolbuffers/protobuf/archive/v3.13.0.tar.gz
+  3.15.4:
+    sha256: "Bad SHA that I need to fix"
```

Comment:
```suggestion
    sha256: 07f8a02afc14a657f727ed89a8ec5627b9ecc47116d60acaabaa1da233bd2e8f
```

Reply:
Thanks!

I will probably wait for 3.15.5, unless someone else asks for it, because my fix landed in 3.15.4 and it kinda breaks Protobuf (RTTI is disabled by default due to a mistake on my part). It is not a big deal, but it doesn't do Conan any favor if it publishes a version that is notoriously broken :)

I think we don't want to maintain too many versions here, so I suppose it's fine to wait one more week.

Am I missing something?

Reply:
No, you're right and I think it's ok to wait for a week.

Reply:
Please mark it as a draft in the meantime so we slip reviewing it

Reply:
Done!